ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -183,18 +174,64 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
     }
 
     // Reference temperature
-    const auto& reference_temperature =
+    double const reference_temperature =
         //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__reference_temperature}
         config.getConfigParameter<double>(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
+    DBUG("Use '%s' as reference temperature.", "reference_temperature");
```

Comment:
You may want to output the value here?

```suggestion
    DBUG("Use '%d' as reference temperature.", reference_temperature);
```